pkg/karmadactl/register/register.go

```diff
@@ -547,35 +890,44 @@ func (o *CommandRegisterOption) constructKarmadaAgentConfig(bootstrapClient *kub
 	if err != nil {
 		return nil, err
 	}
-
-	klog.V(1).Infof("Waiting for the client certificate to be issued")
+	klog.V(1).Infof(fmt.Sprintf("Waiting for the client certificate %s to be issued", csrName))
 	err = wait.PollUntilContextTimeout(context.TODO(), 1*time.Second, o.Timeout, false, func(context.Context) (done bool, err error) {
 		csrOK, err := bootstrapClient.CertificatesV1().CertificateSigningRequests().Get(context.TODO(), csrName, metav1.GetOptions{})
 		if err != nil {
-			return false, fmt.Errorf("failed to get the cluster csr %s. err: %v", o.ClusterName, err)
+			return false, fmt.Errorf("failed to get the cluster csr %s. err: %v", csrName, err)
 		}
 
 		if csrOK.Status.Certificate != nil {
-			klog.V(1).Infof("Signing certificate successfully")
+			klog.V(1).Infof(fmt.Sprintf("Signing certificate of csr %s successfully", csrName))
 			cert = csrOK.Status.Certificate
 			return true, nil
 		}
 
-		klog.V(1).Infof("Waiting for the client certificate to be issued")
+		klog.V(1).Infof(fmt.Sprintf("Waiting for the client certificate of csr %s to be issued", csrName))
```

Comment:
```suggestion
		klog.V(1).Infof(fmt.Sprintf("Waiting for the client certificate of csr %s to be issued", csrName))
		klog.V(1).Infof("Approve the CSR %s manually by executing `kubectl certificate approve %s` on the control plane", csrName, csrName)
```
I am preparing to add a log to remind of manually approving the CSR. Is this message accurate?
Effect as follow:
```bash
I1118 10:54:30.535907 1570912 register.go:534] [discovery] writing CA certificate at /etc/karmada/pki/ca.crt
I1118 10:54:33.202828 1570912 register.go:893] Waiting for the client certificate agent-rbac-generator-member4mt7jq to be issued
I1118 10:54:34.206254 1570912 register.go:906] Waiting for the client certificate of csr agent-rbac-generator-member4mt7jq to be issued
I1118 10:54:34.206278 1570912 register.go:907] Approve the CSR agent-rbac-generator-member4mt7jq manually by executing `kubectl certificate approve agent-rbac-generator-member4mt7jq` on the control plane
I1118 10:54:35.207130 1570912 register.go:906] Waiting for the client certificate of csr agent-rbac-generator-member4mt7jq to be issued
I1118 10:54:35.207155 1570912 register.go:907] Approve the CSR agent-rbac-generator-member4mt7jq manually by executing `kubectl certificate approve agent-rbac-generator-member4mt7jq` on the control plane
I1118 10:54:36.207719 1570912 register.go:906] Waiting for the client certificate of csr agent-rbac-generator-member4mt7jq to be issued
I1118 10:54:36.207742 1570912 register.go:907] Approve the CSR agent-rbac-generator-member4mt7jq manually by executing `kubectl certificate approve agent-rbac-generator-member4mt7jq` on the control plane
```